### PR TITLE
Add support for nightly and weekly run

### PR DESCRIPTION
**Pull request by llxia**
Different groups of test builds will be triggers depending on BUILD_TYPE value (nightly, weekly, or release)

related: automation/issues/53

### CODE REVIEW

**Review by LongyuZhang — APPROVED** [review]
LGTM

File: buildenv/jenkins/config/openj9/nightly/jdk8.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "TEST_FLAG" : "NONE", 
+        "PLATFORM_TARGETS" : [
+            { "aarch64_linux" : "defaultTestTargets" },
+            { "aarch64_mac" : "defaultTestTargets" },
```
Comment (JasonFengJ9) [inline buildenv/jenkins/config/openj9/nightly/jdk8.json:6]: Java 8 doesn't support `aarch64_mac`.

File: buildenv/jenkins/config/openj9/weekly/jdk8.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+[
+    {
+        "TEST_FLAG" : "NONE", 
+        "PLATFORM_TARGETS" : [
+            { "aarch64_linux" : "defaultTestTargets,dev.functional" },
+            { "aarch64_mac" : "defaultTestTargets" },
```
Comment (JasonFengJ9) [inline buildenv/jenkins/config/openj9/weekly/jdk8.json:6]: Similar comment here

**Comment by llxia** [comment]
aarch64_mac is removed from jdk8.

**Review by JasonFengJ9 — APPROVED** [review]

**Pull request merged**